src/EnergyPlus/DataDaylighting.hh

```diff
@@ -255,47 +259,23 @@ namespace Dayltg {
         bool HeaderXLineLengthNeeded = true; // X header will likely be the longest line in the file
         int HeaderXLineLength = 0;           // actual length of this X header line
         std::string pointsHeader;            // part of the header that lists the reference points in the same zone
-    };
 
-    struct MapCalcData
-    {
-        // Members
         int TotalMapRefPoints = 0;                 // Number of illuminance map reference points in this zone (up to 100)
-        int zoneIndex = 0;                         // Pointer to zone being mapped
-        int enclIndex = 0;                         // Index to enclosure for this map
-        Array1D<Vector3<Real64>> MapRefPtAbsCoord; // X,Y,Z coordinates of all illuminance map reference points
-        // in absolute coordinate system (m)
-        // Points 1 and 2 are the control reference points
-        Array1D_bool MapRefPtInBounds;    // True when coordinates are in bounds of zone coordinates
-        Array1D<Real64> DaylIllumAtMapPt; // Daylight illuminance at illuminance map points (lux)
-        // following Hr - report avg hr
-        Array1D<Real64> DaylIllumAtMapPtHr;                                                // Daylight illuminance at illuminance map points (lux)
-        Array2D<std::array<Real64, (int)DataSurfaces::WinCover::Num>> IllumFromWinAtMapPt; // (Number of Zones, Total Map Reference Points)
+        Array1D<DaylMapPt> refPts;    
```

Comment:
In a future branch I am going to try to unify normal ref-points with IllumMap ref-points.